recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ build:
     - COV_THRESHOLD=72  # [win]
     - COV_THRESHOLD=73  # [(linux and not ppc64le) or osx]
     # 2022/4/1: Tests fail on ppc64le with total coverage: 63.63%
-    - COV_THRESHOLD=63  # [linux and ppc64le]
+    - COV_THRESHOLD=50  # [linux and ppc64le]
```

Comment:
Can you lower the threshold for all?
i.e. remove the selector.

Reply:
@cbouss Let me know if my most recent change works for you.

Reply:
that works but you might as well replace all lines by this (no selectors anymore):
`- COV_THRESHOLD=50`